main.cpp

```diff
@@ -92,6 +92,72 @@ good to go!
  */
 
 #include <iostream>
+
+
+class FloatType
```

Comment:
use `struct` not `class`, so you don't have to type `public` all the time
- [ ] revise